esp/esp/users/forms/user_profile.py

```diff
@@ -310,6 +310,14 @@ def clean(self):
 StudentInfoForm.base_fields['studentrep_expl'].widget.attrs['rows'] = 8
 StudentInfoForm.base_fields['studentrep_expl'].widget.attrs['cols'] = 45
 
+affiliationChoices = (
+    ('Pick', 'Pick one...'),
```

Comment:
I couldn't get initial= to work properly.  (I think DropdownOtherField would need to be modified to pass the keyword args properly.)  Instead, I followed the example of the other uses of DropdownOtherField , where the "Other" option comes first, so then we don't need the "Pick one" anymore.